subworkflows/nf-core/bed_scatter_bedtools/main.nf

```diff
@@ -18,7 +18,7 @@ workflow BED_SCATTER_BEDTOOLS {
         .map(
             { meta, beds ->
                 // Checks if the scatter count corresponds to the amount of files created. (This doesn't match in some edge cases)
```

Comment:
Not essential, but I think the auto formatter doesn't cope with comments inside operators like this, so maybe worth moving them outside.

Reply:
Done !